ui/components/ui/logo/logo-coinbasepay.scss

```diff
@@ -0,0 +1,3 @@
+[data-theme=dark] .logo-coinbasepay path {
+    fill: var(--color-text-default);
```

Comment:
The prop value given to `LogoCoinbasePay` is `var(--color-text-default)`, so is this rule still needed?

Reply:
Just confirmed this is still required!